src/pages/deploy_application/configure_source.jsx

```diff
@@ -47,6 +47,7 @@ export default function ConfigureSourcePage({
   const uploadCodeFieldRef = useRef(null);
   const tarballRef = useRef(null);
   const dockerfileModalDisclosure = useDisclosure();
+  const [inputField, setInputField] = useState({url: ""})
```

Comment:
Featureful, plugin-extensible IDE for C/C++ and other programming languages

Reply:
No problem !
Take a look to this blog: https://bobbyhadz.com/blog/react-set-input-value-ref#:~:text=import%20%7BuseRef%7D%20from%20'react,%F0%9F%91%87%EF%B8%8F%20access%20input%20value%20console
I hope it will solve the issue

Reply:
@avelynhc 
No problem !
Take a look to this blog: https://bobbyhadz.com/blog/react-set-input-value-ref#:~:text=import%20%7BuseRef%7D%20from%20'react,%F0%9F%91%87%EF%B8%8F%20access%20input%20value%20console
I hope it will solve the issue

Reply:
@tanmoysrt Thanks! it helped.